### PR TITLE
#166816101 add logout route

### PR DESCRIPTION
**Pull request by tunedev**
#### What does this PR do?
It adds Endpoint GET: /users/logout

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
The pivotal [#166816101](https://www.pivotaltracker.com/story/show/166816101)

#### Screenshots (if appropriate)
NA

#### Questions:
NA

### CODE REVIEW

**Review by devPinheiro — APPROVED** [review]
It looks good to me

File: package.json [hunk 1]
```diff
@@ -47,7 +49,7 @@
   },
   "devDependencies": {
     "coveralls": "^3.0.4",
-    "eslint": "^6.0.0",
+    "eslint": "^5.3.0",
```
Comment (fxola) [inline package.json]: Nice work @swisskid95, Is there any particular reason you chose to update the existing version of eslint to a lower one?

Comment (tunedev) [reply package.json]: Yeah, I got an error that the version of the config-airbnb-base is not peer-dependency with the eslint version originally specified. it suggested a downgrade to this particular version.

Comment (fxola) [reply package.json]: Oh okay. cool

**Pull request closed without merging**